catalogue.html

```diff
@@ -191,6 +198,15 @@ <h2>Доп. Информация о расположении и контакты
       <a href="https://htmlacademy.ru">HTML Academy</a>
     </p>
   </footer>
+  <form class="#" action="#" method="post">
+    <button type="button">#</button>
+    <!--Кнопка закрыть-->
+    <p>
+      Товар добавлен в корзину!
+    </p>
+  <input type="button" value="Оформить заказ"  />
```

Comment:
```suggestion
  <button type="button">Оформить заказ</button>
```
Вместо инпутов используй кнопки